x-pack/plugins/beats/wallaby.js

```diff
@@ -3,15 +3,18 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+const path = require('path');
```

Comment:
Just a suggestion but it may be beneficial from an organizational perspective to include these changes in a separate PR as they don't really have bearing on the rest of the changes we're reviewing here.

Reply:
Hm - that does make sense; it crossed my mind several times while reviewing this that there are portions that could be broken out into separate merges, but like you said, it's a new project and not maintenance on an existing project, so I think if we were to break this off we'd want to do the same for the rest of this PR. From that perspective, I'm ok to keep this in here.